mvp.feature

```diff
@@ -60,54 +60,54 @@ Feature: User with no Author identity in the Authority Register for Personas see
+      | Alternative title(s)             |
+      | Abstract                         |
+      | Alternative abstract(s)          |
+      | Description                      |
+      | Publication date                 |
+      | NPI-fagfelt                      |
+      | Keywords                         |
+      | Primary language for publication |
     And they they see that the Reference tab is pre-populated with suggested metadata values (if available in the input data from the link) for
-        | Publication type |
-        | Link (the link that was provided by the user) |
-        | The unmapped text for Journal |
-        | Volume |
-        | Issue |
-        | Page from | 
-        | Page to |
-        | Article number |
-        | Peer-review |          
+      | Publication type                              |
+      | Link (the link that was provided by the user) |
+      | The unmapped text for Journal                 |
+      | Volume                                        |
+      | Issue                                         |
+      | Page from                                     |
+      | Page to                                       |
+      | Article number                                |
+      | Peer-review                                   |
     And they they see that the Contributors tab Authors section is pre-populated with suggested metadata values (if available in the input data from the link) for <Author>, <Institution> and <CorrespondingAuthor>
     And they they see that the Contributors tab Contributors section is pre-populated with suggested metadata values (if available in the input data from the link) for <ContributionType>, <Name> and <Institution>
     And they they see that the Files and Licenses tab is pre-populated with suggested metadata values (if available in the input data from the link) for
-        | Possible mapped value for License, Archiving policy and Unit agreement |
-        | Possible file for upload  (Filename, File size) |
-	And the Publication is saved and the title is listed in My Publications and marked as Kladd
-    
-        Examples:
-            | Author | Institution | CorrespondingAuthor |
-            | Name Nameson (Not found) | NTNU (Found) | True |
-            | Jan Jansson (Found) | MTNU (Not Found) | False |
-        
-        Examples:
-            | ContributionType | Name | Institution |
-            | Photographer | Jim Jimsson | AVH (Not found) |            
+      | Possible mapped value for License, Archiving policy and Unit agreement |
+      | Possible file for upload  (Filename, File size)                        |
+    And the Publication is saved and the title is listed in My Publications and marked as Kladd
+
+    Examples:
+      | Author                   | Institution      | CorrespondingAuthor |
+      | Name Nameson (Not found) | NTNU (Found)     | True                |
+      | Jan Jansson (Found)      | MTNU (Not Found) | False               |
```

Comment:
Hva betyr det at de er Found/Not Found? En entry av en forfatter blir vel totalt Found/Not Found?